NEWS.rst

```diff
@@ -26,6 +26,11 @@ Removals
     (formerly `take-while`), `tee`, `zip-longest`
   * From `functools`: `reduce`
   * From `fractions`: `Fraction` (formerly `fraction`)
+* Several other core functions have been removed:
+
+  * `instance?`: Use `isinstance` instead.
+  * `keyword`: Use `(hy.models.Keyword (hy.mangle …))` instead.
```

Comment:
No call to `mangle`; Keyword stores the unmangled name

Reply:
I got this backwards. I was gonna say the `keyword` function mangled, but it actually unmangled.